spec/index.bs

```diff
@@ -873,6 +927,103 @@ interface GPUBufferUsage {
 
 ## Buffer Mapping ## {#buffer-mapping}
 
+An application can request to map a {{GPUBuffer}} to get its mapping which is
+an {{ArrayBuffer}} representing the {{GPUBuffer}}'s allocation. Mappings are
+requested asynchronously so that the user agent can ensure the GPU is finished
+using the {{GPUBuffer}} before the application gets its mapping. Mappings can
+be requested for reading with {{GPUBuffer/mapReadAsync}} or writing with
+{{GPUBuffer/mapWriteAsync}}. A mapped {{GPUBuffer}} cannot be used by the GPU
```

Comment:
We should be more specific: instead of saying "used by" we should clarify that we are talking about `submit()` commands involving the buffer and not anything else (e.g. we are not talking about recording). Alternatively, since the timelines already describe that distinction, we can link to the appropriate timeline here (i.e. cannot be used on the XXX timeline).